transport/protocol/src/ticket_aggregation/processor.rs

```diff
@@ -120,7 +120,7 @@ where
         let mut awaiter = self.writer.clone().aggregate_tickets(channel, prerequisites)?;
 
         if let Err(e) = awaiter.consume_and_wait(self.agg_timeout).await {
-            error!("Error occured on ticket aggregation for '{channel}', performing a rollback: {e}");
+            warn!("Error occured on ticket aggregation for '{channel}', performing a rollback: {e}");
```

Comment:
Because it really is not an error on this level. It is an error in the transport layer, where it is detected, it is a warning that a rollback is happening at this layer.